caffe2/ideep/operators/sigmoid_op.cc

```diff
@@ -0,0 +1,60 @@
+#include <caffe2/ideep/ideep_utils.h>
+
+namespace caffe2 {
+
+class IDEEPSigmoidOp final : public IDEEPOperator {
```

Comment:
Please put these classes in anonymous namespace.

Reply:
Hi, I found the other ops (e.g. caffe2/ideep/operators/pool_op.cc) are all put the classes in caffe2 namespace. May I know why this sigmoidop should be in anonymous namespace?